registration/scripts/directives.js

```diff
@@ -56,7 +63,13 @@
                             if (angular.isDefined(deferred)) {
                                 deferred.notify('reject', response);
                             }
+                            apiError = true;
                         });
+                    } else {
+                        //check there are no other error messages on the screen from the api
+                        if (!apiError) {
+                            scope.emptyEmail = true;
+                        }
```

Comment:
I think its best to ignore this as you can set it from the api??

On 24 March 2015 at 11:28, Henry Blyth <notifications@github.com> wrote:

> In registration/scripts/directives.js
> <https://github.com/incuna/angular-registration/pull/15#discussion_r27019274>
> :
>
> >                          });
> > +                    } else {
> > +                        //check there are no other error messages on the screen from the api
> > +                        if (!apiError) {
> > +                            scope.emptyEmail = true;
> > +                        }
>
> emptyEmail feels too specific. Instead could it be that the email is
> required, and when not filled in it's validation will fail and so create
> field errors? I think it should be handled that way, otherwise what if you
> need to mark other fields as required and error when they're empty?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/incuna/angular-registration/pull/15/files#r27019274>.
>